src/api/Files.ts

```diff
@@ -47,7 +46,7 @@ export class Files {
             contents = fs.readdirSync(dir);
         } catch (e) {
             throw new ImperativeError({
-                msg: `An error occurred obtainined the directory info: ${e.message}`,
+                msg: `An error occurred obtained the directory info: ${e.message}`,
```

Comment:
Thanks for fixing the typo 🙂 The grammar still seems off here - should it say "An error occurred obtaining"?